### PR TITLE
PLATUI-2824: Migrate accessibility-statement-frontend to ui-test-runner

### PR DESCRIPTION
**Pull request by kyle-bowden**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -13,14 +13,15 @@ object AppDependencies {
...
   )
 
   val test = Seq(
     "uk.gov.hmrc"       %% s"bootstrap-test-$playVersion" % bootstrapVersion % "test",
-    "org.scalatestplus" %% "selenium-4-2"                 % "3.2.13.0"       % "test",
+    "org.scalatestplus" %% "selenium-4-12"                % "3.2.17.0"       % "test",
     "org.mockito"       %% "mockito-scala-scalatest"      % "1.17.30"        % "test",
     "org.jsoup"          % "jsoup"                        % "1.10.2"         % "test",
-    "uk.gov.hmrc"       %% "webdriver-factory"            % "0.41.0"         % "test"
+    "uk.gov.hmrc"       %% "webdriver-factory"            % "0.46.0"         % "test"
```
Comment (oscarduignan) [inline project/AppDependencies.scala:25]: @kyle-bowden we should be able to remove webdriver-factory now - ui-test-runner replaces it

File: run_acceptance_tests.sh [hunk 2]
```diff
@@ -1,25 +1,6 @@
...
-  -p 5900:5900 \
-  -e PORT_MAPPINGS="$port_mappings" \
-  -e TARGET_IP='host.docker.internal' \
-  ${IMAGE}
-
-sbt \
-  -Dbrowser=$BROWSER \
-  -Denvironment=$ENV \
-  acceptance:test
+sbt clean -Dbrowser="${BROWSER:=chrome}" -Denvironment="${ENVIRONMENT:=local}" "acceptance:test" testReport
```
Comment (oscarduignan) [inline run_acceptance_tests.sh:6]: I reckon we should put a note in docs in this project linking to how to start browser, or even as a comment in here

File: project/plugins.sbt [hunk 3]
```diff
@@ -16,3 +16,5 @@ addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.0")
 addSbtPlugin("com.typesafe.sbt" % "sbt-gzip" % "1.0.2")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-accessibility-linter" % "0.37.0")
```
Comment (ellamdav) [inline project/plugins.sbt:18]: this can be bumped to 0.39.0 to get rid of the Play 2.8 bobby warning

File: project/AppDependencies.scala [hunk 4]
```diff
@@ -13,14 +13,14 @@ object AppDependencies {
     "io.circe"    %% "circe-generic"                    % circeVersion,
     "io.circe"    %% "circe-parser"                     % circeVersion,
     "io.circe"    %% "circe-yaml"                       % circeVersion,
-    "io.circe"    %% "circe-generic-extras"             % circeVersion
+    "io.circe"    %% "circe-generic-extras"             % circeVersion,
+    "uk.gov.hmrc" %% "ui-test-runner"                   % "0.17.0"
```
Comment (ellamdav) [inline project/AppDependencies.scala:17]: per PR-commenter 🤖 this should be scoped to `test` ?

**Review by oscarduignan — APPROVED** [review]

**Pull request merged**